src/rules/add-roles-from-sqlserver.js

```diff
@@ -11,7 +11,7 @@
  */
 
 function addRolesFromSqlServer(user, context, callback) {
-  const tedious = require('tedious');
+  const tedious = require('tedious@6.6.0');
```

Comment:
@shawnmclean I added the explicit version reference here so now it should actually work in Node 8 or Node 12.